react-version/src/style/Game.scss

```diff
@@ -0,0 +1,260 @@
+.game {
+	display: flex;
+	justify-content: center;
+	padding: 20px 0 0 0;
+	transition: 1s;
+
+	.click-start-hex {
+		position: absolute;
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		top: 600px;
+		height: 200px;
+		width: 400px;
+		background-color: #fff;
+		border: 2px solid black;
+		border-radius: 15px;
+		text-align: center;
+		z-index: 1;
+
+		h2 {
+			position: absolute;
+		}
+	}
+}
+
+// plansza
+
+.board-bg {
+	position: absolute;
+	background-image: url("../../../src/img/Stay Alive 2bg.png");
+	background-size: cover;
+	height: 850px;
+	width: 682px;
+}
+
+.middle-container {
+	background-image: url("../../../src/img/Stay Alive 3.png");
+	background-size: cover;
+	height: 850px;
+	width: 682px;
+
+	.copyright-container {
+		display: flex;
+		flex-direction: column;
+		justify-content: flex-start;
+		align-items: center;
+
+		span {
+			font-size: 15px;
+		}
+
+		.copyright-text {
+			letter-spacing: 2px;
+		}
+	}
+
+	button {
+		cursor: pointer;
+	}
+
+	.game-board {
+		display: flex;
+		flex-direction: column;
+		justify-content: flex-end;
+		height: 850px;
+		width: 682px;
+
+		.row1-container {
+			display: flex;
+			justify-content: flex-end;
+			align-items: flex-end;
+			height: 85px;
+			width: 682px;
+
+			.btn-line1 {
+				width: 54px;
+				height: 54px;
+				margin-left: 1.35px;
+				margin-bottom: 37.5px;
+				border: none;
+				background: none;
+			}
+
+			.btn11 {
+				margin-right: 37.5px;
+			}
+
+			.btn-line1:hover {
+				opacity: 30%;
+				background-color: #fff;
+				border-radius: 50px;
+				box-shadow: 0 0 10px 5px white;
+				animation: none;
+			}
+		}
+
+		.row3-container,
+		.row5-container,
+		.row7-container,
+		.row9-container,
+		.row11-container,
+		.row13-container,
+		.row15-container {
+			display: flex;
+			justify-content: flex-end;
+			align-items: flex-end;
+			height: 48px;
+			width: 682px;
+
+			.btn-line3,
+			.btn-line5,
+			.btn-line7,
+			.btn-line9,
+			.btn-line11,
+			.btn-line13,
+			.btn-line15 {
```

Comment:
The same in here, it should be single class. It's a little bit mess with current approach.